### ISSUE TITLE
Wish: one more font size setting

### ISSUE THREAD

**Issue by Spielmops** [moezbhatti/qksms#1854]
I wish, that QKSMS would have a setting for the font size "very big", so that I can chose "small", "normal", "big", "bigger" and "very big". The "bigger"- size is not even as big as the Android-settings-items. If I chose a bigger system font, then fonts everywhere else are to big.

Spielmops

**Comment by dhjaber** [comment]
hi there ,,
look there is an option in settings called use system font 
then you can customize your system font as you like 
=========================================

**Comment by Spielmops** [comment]
My system-font is big enough, but QKSMS could use some font-size more, the size of the messages is small, so there is space on the screen for that. Should I really set my system-font bigger, only for reading a verification-number from Paypal?

Spielmops

**Comment by pyrite357** [comment]
I agree, and was thinking making it a number like font size with a slider might be better for everyone than canned sizes. Just showing support for this feature request. Thank you
